runtime/flashbox/src/lib.rs

```diff
@@ -1828,6 +1828,12 @@ impl_runtime_apis! {
             TransactionPayment::length_to_fee(length)
         }
     }
+
```

Comment:
AFAIK we can now remove the aura-runtime-api and all its requirements from our runtime with this

Reply:
but we can leave it for a separate PR

Reply:
I prefer not to